frontend/apps/erd-web/app/erd/p/[...slug]/page.tsx

```diff
@@ -170,7 +181,9 @@ export default async function Page({
         errorObjects={[
           {
             name: 'NetworkError',
-            message: 'We could not detect the format of the file',
+            message: weCannotAccess,
+            instruction:
+              'Please specify the format in the URL query parameter `format`',
```

Comment:
If possible, I want to add following link: https://liambx.com/docs/web#appendix-schema-format-options
But if it's too much trouble to implement, it doesn't have to be.

Reply:
That's true! However, I think this would require a bit of effort, and there will likely be very few cases where this actually applies. I’m sorry, but I believe it’s more reasonable not to address this for now!

Reply:
@hoshinotsuyoshi I think so too!! Let's proceed as is.